frontend/web/src/contexts/WebSocketContext.tsx

```
@@ -0,0 +1,188 @@
+          const updated = new Map(prev);
+          updated.set(genUpdate.id, genUpdate);
+          return updated;
+        });
+        break;
+
+      case 'queue_update':
+        setQueueStatus(message.payload as QueueUpdate);
+        break;
+
+      case 'model_update':
+        const modelUpdate = message.payload as ModelUpdate;
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)